changelog.d/20221012_190435_sirosen_paginated_flows_list.rst

```diff
@@ -0,0 +1 @@
+* ``FlowsClient.list_flows`` now supports pagination (:pr:`NUMBER`)
```

Comment:
Just to confirm, a hook runs on GH during merge to swap out `` :pr:`NUMBER` ``, right?

Reply:
Yes, it runs after any merge and replaces `NUMBER` with the source PR. The workflow for that is here:
https://github.com/globus/globus-sdk-python/blob/main/.github/workflows/update_pr_references.yaml